packages/tasit-sdk/src/scripts/checkDecentralandContractsData.js

```diff
@@ -0,0 +1,83 @@
+//
+// Script used to check if the target chain has the Decentraland app pre-conditions
+//
+// Note: This script is using mocha for convenince but isn't a test suite to be runned by `test` script.
+// Use: npx mocha src/scripts/checkDecentralandContractsData.js  --require @babel/register --file src/testHelpers/mochaSetup.js --no-timeouts
+//
+
+import { Action, ContractBasedAccount } from "../TasitSdk";
+const { ERC20 } = Action;
+const { Mana } = ERC20;
+
+import DecentralandUtils from "../helpers/DecentralandUtils";
+
+const fetch = require("node-fetch");
+
+const { TWO, TEN, BILLION, WeiPerEther } = constants;
```

Comment:
It feels weird that `WeiPerEther` has a different capitalization convention

Reply:
Done.

Reply:
Thanks! Resolving this.